client/components/select-dropdown/docs/example.jsx

```diff
@@ -101,48 +103,112 @@ const SelectDropdownDemo = React.createClass( {
 					<DropdownItem
 						count={ 3 }
 						selected={ this.state.childSelected === 'Trashed' }
-						onClick={ this.selectItem.bind( this, 'Trashed', 3 ) }
+						onClick={ this.getSelectItemHandler( 'Trashed', 3 ) }
 					>
 						Trashed
 					</DropdownItem>
 				</SelectDropdown>
 
-				<h3 style={ { marginTop: 20 } }>max-width: 220px;</h3>
+				<h3 style={ { marginTop: 20 } }>With Icons in Items Passed as Options</h3>
+				<SelectDropdown
+					compact={ this.state.compactButtons }
+					onSelect={ this.onDropdownSelect }
+					options={ [
+						{ value: 'published', label: 'Published', icon: <Gridicon icon="align-image-left" size={ 18 } /> },
+						{ value: 'scheduled', label: 'Scheduled', icon: <Gridicon icon="calendar" size={ 18 } /> },
+						{ value: 'drafts', label: 'Drafts', icon: <Gridicon icon="create" size={ 18 } /> },
+						{ value: 'trashed', label: 'Trashed', icon: <Gridicon icon="trash" size={ 18 } /> },
+					] }
+				/>
+
+				<h3 style={ { marginTop: 20 } }>With Icons in Items Passed as Children</h3>
+				<SelectDropdown
+					compact={ this.state.compactButtons }
+					onSelect={ this.onDropdownSelect }
+					selectedText={ this.state.childSelected }
+					selectedIcon={ this.state.selectedIcon }
+				>
```

Comment:
Please add `selectedCount`, so that this example doesn't look confusing/broken (no count shown for the selected item).

<img width="399" alt="screencapture at tue jun 6 15 30 45 edt 2017" src="https://user-images.githubusercontent.com/2098816/26848228-b0a6134e-4acd-11e7-89ed-4ed824d864bc.png">